doc/usage.rst

```diff
@@ -101,9 +101,10 @@ Usage::
 
   pyperformance run [-h] [-r] [-f] [--debug-single-value] [-v] [-m]
                        [--affinity CPU_LIST] [-o FILENAME]
-                       [--append FILENAME] [--manifest MANIFEST]
-                       [-b BM_LIST] [--inherit-environ VAR_LIST]
-                       [-p PYTHON]
+                       [--append FILENAME] [--min-time MIN_TIME]
+                       [--same-loops SAME_LOOPS] [--timeout TIMEOUT]
```

Comment:
Could you document `--min-time` and `--same-loops` in a different pull request?

Reply:
`--same-loops` has just moved from the end of the list to the right place. it was misplaced in the first place.
`--min-time` was never added here but it's not new.

Shall I put both of them in a PR?

Reply:
Moving it to a different place is probably fine, but updating the usage should probably be in a separate PR.

Reply:
I implemented `--same-loops` a few months ago -- it's documented in here and in the usage.rst.  Was there something else you are looking for?  I'll volunteer to submit if so (in a separate PR).

Reply:
The changes in usage.rst are there to reflect the actual output of pyperformance. In the PR https://github.com/python/pyperformance/pull/327 implemented the same loops option but it has been forgotten to be added here.